foundry/foundry.py

```diff
@@ -35,9 +35,11 @@
 import logging
 import os
 import shutil
+import warnings
 logging.disable(logging.INFO)
 
 
+
```

Comment:
this is really small, but the whitespace should be adjusted such that there are 2 lines between `import warnings` and `logging.disable()`, and then only 2 lines after `logging.disable()`, not 3